iroha/src/crypto.rs

```diff
@@ -16,24 +17,57 @@ use ursa::{
 
 /// Represents hash of Iroha entities like `Block` or `Transaction.
 pub type Hash = [u8; 32];
+
+/// Pair of Public and Private keys.
+#[derive(Clone)]
+pub struct KeyPair {
+    /// Public Key.
+    pub public_key: PublicKey,
+    /// Private Key.
+    pub private_key: PrivateKey,
+}
+
 /// Public Key used in signatures.
-pub type PublicKey = [u8; 32];
+#[derive(Encode, Decode, PartialEq, Eq, PartialOrd, Debug, Clone, Hash, Default)]
+pub struct PublicKey {
+    origin: [u8; 32],
```

Comment:
Maybe we can use constants for primitive's sizes? Like `PUBLIC_KEY_SIZE`

Reply:
We had such a thing, but I don't think we need to introduce such a constant before https://jira.hyperledger.org/browse/IR-745